tests/test_fits_to_zarr.py

```
@@ -125,10 +135,250 @@ def test_mhz_from_name_no_match():
+
+
+def test_rechunk_lm_for_zarr_uniform_spatial_chunks():
+    """Irregular dask chunks along l/m must become uniform for Zarr compatibility."""
+    import dask.array as da
+    import numpy as np
+    import xarray as xr
+
+    mod = _import_module()
+    arr = np.random.default_rng(0).random((512, 512))
+    data = da.from_array(arr, chunks=((256, 256), (256, 128, 128)))
+    l = np.linspace(-1.0, 1.0, 512)
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)